pkg/servicecentral/types.go

```diff
@@ -30,11 +30,40 @@ type ServiceData struct {
 	Stateless            bool   `json:"stateless,omitempty"`
 	BusinessUnit         string `json:"business_unit,omitempty"`
 
+	Attributes []ServiceAttribute `json:"service_attributes,omitempty"`
+
 	// Compliance is a read-only field. It can be nil, in which case it means
 	// they have not completed their compliance questions yet
 	Compliance *ServiceComplianceConf `json:"compliance,omitempty"`
 }
 
+type ServiceAttribute struct {
+	Team string `json:"team,omitempty"`
+}
+
+type ServiceAttributeResponse struct {
```

Comment:
If/when we have more service attributes it is likely that we will need to change these types.